sway/config.c

```diff
@@ -68,9 +69,19 @@ struct sway_config *read_config(FILE *file, bool is_active) {
 			goto _continue;
 		}
 
-		if (!temp_depth && handle_command(config, line) != true) {
+		// Any command which would require wlc to be initialized
+		// should be queued for later execution
+		list_t *args = split_string(line, " ");
+		if (strcmp("workspace", args->items[0]) == 0) {
+			sway_log(L_DEBUG, "Deferring command %s", line);
+			char *cmd = malloc(strlen(line) + 1);
+			strcpy(cmd, line);
+			list_add(config->cmd_queue, cmd);
+		}else if (!temp_depth && !handle_command(config, line)) {
+			sway_log(L_DEBUG, "Config load failed for line %s", line);
 			success = false;
-		}
+		} 
+		list_free(args);
```

Comment:
Consider how this works when reloading, and also add a space between `}` and `else`